formatTest/unit_tests/expected_output/uncurried.re

```diff
@@ -0,0 +1,115 @@
+f(.);
+
+[@attr]
+f(.);
+
+f(.);
+
+f(. a, b, c);
+
+[@attr]
+f(. a, b, c);
+
+f(. ~a, ~b, ~c);
```

Comment:
currently, uncurried application does not support labels, maybe you can make an error earlier?